build.sh

```diff
@@ -197,7 +197,12 @@ test_ios_static() {
 
     destination="$1"
     xc "-scheme 'Realm iOS static' -configuration $CONFIGURATION -sdk iphonesimulator -destination '$destination' build"
-    xc "-scheme 'Realm iOS static' -configuration $CONFIGURATION -sdk iphonesimulator -destination '$destination' test 'ARCHS=\$(ARCHS_STANDARD_32_BIT)'"
+    if [[ `echo $REALM_XCODE_VERSION | awk '{tmp=match($0, /^[[:digit:]]+/); print substr($0, RSTART, RLENGTH)}'` -lt 9 ]]; then
```

Comment:
It's hard to tell precisely what this is doing. My assumption is that it's extracting the first digit of the Xcode version number? If so, `${REALM_XCODE_VERSION%%.*}` would be a more concise way of going about it (that expands to `REALM_XCODE_VERSION` with the the longest substring matching `.*` removed from the end of it). I'd also be inclined to wrap this up in a helper function, since we appear to perform this same test elsewhere. That'd let us write this comparison more like:

```sh
if (( xcode_version_major < 9 )); then
fi
```